capsules/src/seven_segment.rs

```diff
@@ -0,0 +1,397 @@
+//! Provides userspace access to 7 segment digit displays.
+//!
+//! This capsule was developed using the following components:
+//! - Microbit_v2
+//! - Edge Connector Breakout Board for Microbit
+//! - 7 segment display with 4 digits
+//! - breadboard, 220 ohms resistances and jump wires
+//!
+//! Usage
+//! -----
+//!
+//! Cannot use simultaneously with LED matrix.
```

Comment:
Why not?

Reply:
This was an error. I meant to say that it cannot be used simultaneously with LED matrix on the Microbit because of lack of pins. I will remove this statement.